pkgs/data/fonts/intel-one-mono/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenvNoCC, fetchurl, unzip}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "intel-one-mono";
+  version = "1.2.0";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/otf.zip";
+      sha256 = "sha256-VnXIaW77dRXvXB1Vr01xRQDMECltwzF/RMqGgAWnu5M=";
+     })
+    (fetchurl {
+      url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/ttf.zip";
+      sha256 = "sha256-kaz0DePeO8nvKVonYJhs5f2+ps/5Xo5psjg1hoxzaiU=";
+    })
+  ];
+
+  # Multiple packages are download in srcs folder
+  # -> need to specify where the unpack phase will `cd` into
+  # use the `srcs` folder as the Root of the unpack phase
+  sourceRoot = ".";
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  doCheck = false;
+  dontFixup = true;
+
+  # `unzip` is not in the standard environment, so it must be added to
+  # nativeBuildInputs when the source files are `.zip`
```

Comment:
```suggestion
```

Reply:
Is it necessary to remove comments? I find them usefull for newcomers that look into derivations to understand how they work

Reply:
Such a comment would be fitting in a guide or tutorial, not one of over 80k packages. Also the line of code is self explanatory: if the package would be available then we wouldn't need to add it and unzip can only really do one thing.